### ISSUE TITLE
Zuul Job release-ansible-collection-galaxy failing for new Ansible Galaxy

### ISSUE THREAD

**Issue by sean-freeman** [ansible/zuul-config#526]
## Zuul Job release-ansible-collection-galaxy failing for new Ansible Galaxy

Upon new release created for Ansible Collection, the Zuul Job `release-ansible-collection-galaxy` has an issue with the latest Ansible Galaxy website release pushed live this weekend.

> Task Summary
`controller:`
```
❌ Task  Publish collection to Ansible Galaxy / Automation Hub  failed running on host  controller

/home/zuul/src/github.com/ansible-network/releases/.tox/generate_collection_version/bin/ansible-galaxy -vvv collection publish /home/zuul/artifacts/xxxx.tar.gz

non-zero return code

ERROR! Error when finding available api versions from release_galaxy (https://galaxy.ansible.com/) (HTTP Code: 403, Message: Forbidden)
```

**Comment by sean-freeman** [comment]
@morucci @Andersson007 @alinabuzachis @jillr As active maintainers, can you please review? This issue will stop all Ansible Collection releases from triggering

**Comment by jillr** [comment]
Looking into this now; I think we may just need to upgrade the version of ansible-core used by the publishing job but we will confirm

**Comment by jillr** [comment]
@sean-freeman Could you please try publishing again?  The error is looking more like a credentials issue (maybe the cutover expired our API key?) and we had been planning a key rotation anyway so I've just proceeded with that key rotation.

**Comment by jillr** [comment]
unfortunately I've been unable to reach anyone who can directly log in to the github user the token we use is attached to.  I should be able to reach them tomorrow, apologies for the delay!

**Comment by jillr** [comment]
We've determined we have an issue with namespace access to our release bot user.  We're not tracking this on the forum.  https://forum.ansible.com/t/bot-used-for-publishing-has-lost-namespace-access/1413

**Comment by jillr** [comment]
@sean-freeman We have a work-around.  Is the tarball from https://github.com/sap-linuxlab/community.sap_install/releases/tag/1.3.2 the one you need uploaded to Galaxy?  I can get that or any other missing collection releases since Sept 30, that are in the community namespace, manually published for you while we work out the issue.

**Comment by sean-freeman** [comment]
@jillr That is the correct/current release that needs to be published to Ansible Galaxy; then my project's requirements would be fulfilled.

However, I am not the only one as I can see AWS is also caught in the queue and obviously the longer it continues the more manual effort to resolve the queue.

**Comment by jillr** [comment]
amazon.aws is already on the list for manual publishing - we'll get you taken care of!

**Comment by jillr** [comment]
https://galaxy.ansible.com/ui/repo/published/community/sap_install 1.3.2 has been uploaded to Galaxy and is available for users.  We have some more work to do on the pipeline, in the interim if you need to make any releases before that's sorted out please just ping me!

Thanks so much for your patience, @sean-freeman.

**Comment by jillr** [comment]
Release pipelines are confirmed working again
